src/backend/storeManagers/sideload/games.ts

```diff
@@ -128,6 +129,7 @@ export async function uninstall({
   notify({ title, body: i18next.t('notify.uninstalled') })
 
   removeShortcutsUtil(gameInfo)
+  removeRecentGame(appName)
```

Comment:
could you add awaits and `await removeNonSteamGame({ gameInfo })` as well ?

Reply:
I understand we intentionally don't await for these things, they can happen async and makes it faster

Reply:
Could still wrap 'em in an `await Promise.all([ ... ])` to at least run them in parallel

Reply:
aren't those already running in parallel? I think we really don't care about waiting for the result of those calls at all

is there any benefit I'm missing?

Reply:
I thought it would be better if everything was removed by the time this function returned, but it's not necessary